src/main/resources/schema.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TYPE user_role AS ENUM ('CLIENT', 'ADMIN');
+
+CREATE TYPE ticket_type AS ENUM ('DAY', 'WEEK', 'MONTH', 'YEAR');
+
+CREATE TABLE users_info
+(
+    id VARCHAR(36) PRIMARY KEY NOT NULL,
+    name VARCHAR(15) NOT NULL,
+    creation_date date,
+    user_role USER_ROLE NOT NULL DEFAULT 'CLIENT'
+);
+
+CREATE TABLE tickets
+(
+    id VARCHAR(36) PRIMARY KEY NOT NULL,
+    user_id VARCHAR(36)
+    ticket_type ticket_type NOT NULL,
+    creation_date date,
+    FOREIGN KEY (user_id) REFERENCES users_info (id) ON DELETE CASCADE, NOT NULL
+);
```

Comment:
1. missed comma after `user_id VARCHAR(36)`
2. `not null` has to be specified near the column description, not in foreign key description:
```sql
CREATE TABLE tickets
(
    id VARCHAR(36) PRIMARY KEY NOT NULL,
    user_id VARCHAR(36) NOT NULL,
    ticket_type ticket_type NOT NULL,
    creation_date date,
    FOREIGN KEY (user_id) REFERENCES users_info (id) ON DELETE CASCADE 
);
```

Reply:
Fixed